### ISSUE TITLE
"No results found!" on fresh ani-cli install

### ISSUE THREAD

**Issue by CertNZone** [pystardust/ani-cli#1251]
**Metadata (please complete the following information)**
Version: 4.7
OS: Windows 10
Shell: Powershell profile inside Terminal Preview
Anime: all

No matter what anime I search I get "No Results Found". This is a fresh install of ani-cli so can't be the versioning issue that was fixed.

I have attempted to install the dependencies through both scoop and winget and restarted the shell between installs and attempting to run ani-cli.

I'm usnure what else to include as I can't get it to run in the first place, and am receiving no error message.

**Comment by CertNZone** [comment]
I believe I have it working. Sometimes if I enter:
```
> ani-cli
> Checking dependencies...
> Search anime: [insert anime name]
```
I get the no results found issue, but if I enter:
```
> ani-cli [insert anime name]
```
I get the results even if using the same anime. 
When I used "scoop update ani-cli" it said it was up to date and didn't fix anything, but using "ani-cli -U" to update the script it started working. There may still be an underlying issue, but I believe I've got it working enough
